projects/media-viewer/src/lib/viewers/pdf-viewer/side-bar/side-bar.component.ts

```diff
@@ -78,11 +80,19 @@ export class SideBarComponent implements OnInit, OnChanges, OnDestroy {
     const id = uuid();
     bookmark.name = bookmark.name.substr(0,30);
     this.bookmarksStore.dispatch(new CreateBookmark({ ...bookmark, documentId, id }));
-    this.toolbarButtons.sidebarOpen.next(true);
-    this.selectedView = 'bookmark'
+    this.editBookmark(id);
```

Comment:
i know is not your code but maybe we could change this... there is no such thing as bookmarksStore... store is always just one global... so just call it store